blogs/2016-03-07-ci-with-xcode-server-and-fastlane.md

```diff
@@ -481,10 +481,8 @@ $ fastlane before_integration_staging
 ```
 
 On the output log appears next messages:
-
-    security: SecKeychainAddInternetPassword <NULL>: User interaction is not allowed.
+    security: SecKeychainAddInternetPassword `<NULL/>`: User interaction is not allowed.
```

Comment:
The change from <NULL> to <NULL/> is not acceptable. The author meant to write it without the `/`

Reply:
Done